doc/src/build/move/sui-move-library.md

```diff
@@ -23,54 +23,43 @@ transfer::transfer(obj, recipient);
 This call will fully consume the object, making it no longer accessible in the current transaction.
 Once an address owns an object, for any future use (either read or write) of this object, the signer of the transaction must be the owner of the object.
 
-### Transfer to object
-We can also transfer an object to be owned by another object. Note that the ownership is only tracked in Sui. From Move's perspective, these two objects are still more or less independent, in that the child object isn't part of the parent object in terms of data store.
-Once an object is owned by another object, it is required that for any such object referenced in the entry function, its owner must also be one of the argument objects. For instance, if we have a chain of ownership: address `Addr1` owns object `a`, object `a` owns object `b`, and `b` owns object `c`, in order to use object `c` in a Move call, the entry function must also include both `b` and `a`, and the signer of the transaction must be `Addr1`, like this:
-```
-// signer of ctx is Addr1.
-public entry fun entry_function(a: &A, b: &B, c: &mut C, ctx: &mut TxContext);
-```
+### Owned by another object
+
+An object can be owned by another object when the former is added as a [dynamic object field](../programming-with-objects/ch5-dynamic-fields.md) of the latter. Note that the ownership is only tracked in Sui. From Move's perspective, these two objects are still more or less independent in that the child object isn't stored as part of the parent object, but they are associated with each other:
```

Comment:
> From Move's perspective, these two objects are still more or less independent in that the child object isn't stored as part of the parent object, but they are associated with each other:

This comment confuses me a bit. From Move's perspective, I would say the object is just wrapped. There is no "ownership" like there was with the child object API. 
Thoughts?

Reply:
I'll try and generally steer away from the "ownership" terminology.

Reply:
I also want to distinguish this state from being wrapped because we're talking about dynamic object fields and not dynamic fields here.